### PR TITLE
Converting from HTML to Markdown part #6

### PR DESCRIPTION
**Pull request by pangolp**
## Description

The idea of the pull request, is to replace the tables written in HTML, and convert them to markdown format. Links are also created to the attributes, so that they can be accessed, from the table.

## Issues Addressed:
https://github.com/azerothcore/wiki/issues/247

## Links
https://github.com/azerothcore/wiki/search?q=%3Ctbody%3E
https://jmalarcon.github.io/markdowntables/
https://www.tablesgenerator.com/markdown_tables

## List of changes
- [x] docs/instance.md
- [x] docs/character_homebind.md
- [x] docs/playercreateinfo.md
- [x] docs/logs.md
- [x] docs/spell_target_position.md
- [x] docs/character_banned.md
- [x] docs/creature_equip_template.md
- [x] docs/guild_bank_right.md
- [x] docs/instance_encounters.md
- [x] docs/phase_area.md
- [x] docs/pool_creature.md
- [x] docs/creature_model_info.md
- [x] docs/quest_poi.md
- [x] docs/item_set_names_locale.md
- [x] docs/guild_eventlog.md
- [x] docs/player_factionchange_items.md
- [x] docs/channels.md
- [x] docs/pool_gameobject.md
- [x] docs/pool_pool.md
- [x] docs/areatrigger.md
- [x] docs/account_tutorial.md
- [x] docs/game_tele.md
- [x] docs/creature.md
- [x] docs/spell_bonus_data.md
- [x] docs/guild_bank_eventlog.md
- [x] docs/player_levelstats.md
- [x] docs/quest_greeting.md
- [x] docs/achievement_reward.md
- [x] docs/spell_linked_spell.md
- [x] docs/reputation_spillover_template.md
- [x] docs/game_event_npc_vendor.md
- [x] docs/areatrigger_teleport.md
- [x] docs/areatrigger_template.md
- [x] docs/character_battleground_data.md
- [x] docs/lfg_dungeon_rewards.md
- [x] docs/creature_formations.md
- [x] docs/item_effect.md
- [x] docs/pvpstats_players.md
- [x] docs/auctionhouse.md
- [x] docs/points_of_interest.md
- [x] docs/guild.md

### CODE REVIEW

**Comment by pangolp** [comment]
Pull request completed and awaiting review.
@FrancescoBorzi @Helias

File: docs/account_tutorial.md [hunk 1]
```diff
@@ -135,10 +42,8 @@ These values 32bits flags. So 8 x 32bits values makes 256 bits available to stor
 
 Each bit means:
 
-``` cpp
-  0  -  Not yet shown
-  1  -  Shown
-```
+0  -  Not yet shown
+1  -  Shown
```
Comment (Helias) [inline docs/account_tutorial.md:47]: these lines appear one next-to the other, try to put a newline in between, that means in markdown 2 spaces at the end of the first line

File: docs/channels.md [hunk 2]
```diff
@@ -113,28 +36,27 @@ Name of the channel
...
+| Horde    | 67    |
+| Alliance | 469   |
 
 ### announce
 
 **Channel announce (0/1)**
 
 0 = Channel join/part actions will not be sent
-
 1 = Channel join/part actions will be sent
```
Comment (Helias) [inline docs/channels.md:53]: same here

File: docs/channels.md [hunk 3]
```diff
@@ -113,28 +36,27 @@ Name of the channel
...
 
 ### ownership
 
 **Channel ownership (0/1)**
 
 0 = No one will ever be an owner.
-
 1 = Ownership is the first person in the channel.
```
Comment (Helias) [inline docs/channels.md:60]: same here, need new line

**Review by Helias — CHANGES_REQUESTED** [review]

**Comment by Helias** [comment]
This is wonderful

File: docs/creature_formations.md [hunk 4]
```diff
@@ -142,25 +64,8 @@ Sets group member behaviours, values are:
...
-If your leader has a path like the one below where he moves to point 5 then back to 1 you would set point\_1 = 4 and point\_2 = 8 on the memberGUID. If the memberGUID is at angle 90 up to point 5 it will switch to angle 270 for the return trip. This is only needed to keep creatures on the correct side. these values can be left at 0 for creatures following directly behind leaderGUID or any creatures in a circular path. 
-
- 
-
-1     2     3     4    5
-
------&lt;---------&gt;------
-
-       8    7      6
-
```
Comment (Kitzunu) [inline docs/creature_formations.md:154]: Why do you remove this?

File: docs/guild.md [hunk 5]
```diff
@@ -8,150 +8,33 @@ This table holds the main guild information. All created guilds or all guilds in
...
-<td><p>0</p></td>
-<td><p> </p></td>
-<td><p> </p></td>
-</tr>
-</tbody>
-</table>
+| Field                | Type         | Attributes | Key | Null | Default | Extra  | Comment |
+|----------------------|--------------|------------|-----|------|---------|--------|---------|
+| [guildid][1]         | int(10)      | unsigned   | PRI | NO   | 0       | Unique |         |
+| [name][2]            | varchar(24)  | signed     |     | NO   | "       |        |         |
```
Comment (Kitzunu) [inline docs/guild.md:14]: Default is blank, not "

File: docs/guild.md [hunk 6]
```diff
@@ -8,150 +8,33 @@ This table holds the main guild information. All created guilds or all guilds in
...
+| [leaderguid][3]      | int(10)      | unsigned   |     | NO   | 0       |        |         |
+| [EmblemStyle][4]     | tinyint(3)   | unsigned   |     | NO   | 0       |        |         |
+| [EmblemColor][5]     | tinyint(3)   | unsigned   |     | NO   | 0       |        |         |
+| [BorderStyle][6]     | tinyint(3)   | unsigned   |     | NO   | 0       |        |         |
+| [BorderColor][7]     | tinyint(3)   | unsigned   |     | NO   | 0       |        |         |
+| [BackgroundColor][8] | tinyint(3)   | signed     |     | NO   | 0       |        |         |
+| [info][9]            | text         | signed     |     | NO   | "       |        |         |
+| [motd][10]           | varchar(128) | signed     |     | NO   | "       |        |         |
```
Comment (Kitzunu) [inline docs/guild.md:22]: Here too

File: docs/logs.md [hunk 7]
```diff
@@ -97,30 +36,32 @@ The [RealmID](realmlist#id) of the realm this log string came from. 0 if realmd.
...
+| LOG_TYPE_ERROR  | 1           | SOME SORT OF ERROR          |
+| LOG_TYPE_BASIC  | 2           | BASIC INFORMATION           |
+| LOG_TYPE_DETAIL | 3           | DETAILED INFORMATION        |
+| LOG_TYPE_DEBUG  | 4           | DEBUGGING INFORMATION       |
+| LOG_TYPE_CHAR   | 5           | CHARACTER-RELATED           |
+| LOG_TYPE_WORLD  | 6           | WORLD PACKET DUMP           |
+| LOG_TYPE_RA     | 7           | REMOTE ACCESS LOGS          |
+| LOG_TYPE_GM     | 8           | GM COMMAND LOGS             |
+| LOG_TYPE_CRASH  | 9           | CRASH ALARM LOGS            |
+| LOG_TYPE_CHAT   | 10          | CHAT/MESSAGE LOGS           |
```
Comment (Kitzunu) [inline docs/logs.md:51]: If anything Name | Value | Description

File: docs/phase_area.md [hunk 8]
```diff
@@ -11,50 +11,15 @@ Each line holds one Zone or Area with one phaseId and a description.  Each line
...
-</td>
-<td><br />
-</td>
-</tr>
-</tbody>
-</table>
+| Field        | Type    | Null | Key | Default | Extra |
+|--------------|---------|------|-----|---------|-------|
+| [AreaId][1]  | int     | No   | Pri | 0       |       |
+| [PhaseId][2] | int     | No   | Pri | 0       |       |
```
Comment (Kitzunu) [inline docs/phase_area.md:17]: capitalize no and pri

File: docs/points_of_interest.md [hunk 9]
```diff
@@ -8,110 +8,23 @@
...
-</table>
+| Field           | Type         | Attributes | Key | Null | Default | Extra | Comment |
+|-----------------|--------------|------------|-----|------|---------|-------|---------|
+| [entry][1]      | mediumint(8) | unsigned   | PRI | NO   | 0       |       |         |
+| [x][2]          | float        | signed     |     | NO   | 0       |       |         |
+| [y][3]          | float        | signed     |     | NO   | 0       |       |         |
+| [icon][4]       | mediumint(8) | unsigned   |     | NO   | 0       |       |         |
+| [flags][5]      | mediumint(8) | unsigned   |     | NO   | 0       |       |         |
+| [Importance][6] | mediumint(8) | unsigned   |     | NO   | 0       |       |         |
+| [icon_name][7]  | text         | signed     |     | NO   | NULL    |       |         |
```
Comment (Kitzunu) [inline docs/points_of_interest.md:19]: ID
PositionX
PositionY
Icon
Flags
Importance
Name -> Text so not signed or unsigned. and default nothing.

File: docs/pool_creature.md [hunk 10]
```diff
@@ -8,70 +8,17 @@ This table contains a list of creatures that are tied to a specific pool.
...
-<td><p> </p></td>
-</tr>
-</tbody>
-</table>
+| Field            | Type         | Attributes | Key | Null | Default | Extra | Comment |
+|------------------|--------------|------------|-----|------|---------|-------|---------|
+| [guid][1]        | int(10)      | unsigned   | PRI | NO   | 0       |       |         |
+| [pool_entry][2]  | mediumint(8) | unsigned   |     | NO   | 0       |       |         |
+| [chance][3]      | float        | unsigned   |     | NO   | 0       |       |         |
+| [description][4] | varchar(255) | signed     |     | YES  | NULL    |       |         |
```
Comment (Kitzunu) [inline docs/pool_creature.md:16]: can't be signed or unsigned because it is varchar

File: docs/pool_gameobject.md [hunk 11]
```diff
@@ -9,70 +9,17 @@ This table can only contain gameobjects that have a type of GAMEOBJECT\_TYPE\_CH
...
-<td><p> </p></td>
-</tr>
-</tbody>
-</table>
+| Field            | Type         | Attributes | Key | Null | Default | Extra | Comment |
+|------------------|--------------|------------|-----|------|---------|-------|---------|
+| [guid][1]        | int(10)      | unsigned   | PRI | NO   | 0       |       |         |
+| [pool_entry][2]  | mediumint(8) | unsigned   |     | NO   | 0       |       |         |
+| [chance][3]      | float        | unsigned   |     | NO   | 0       |       |         |
+| [description][4] | varchar(255) | signed     |     | YES  | NULL    |       |         |
```
Comment (Kitzunu) [inline docs/pool_gameobject.md:17]: same

File: docs/spell_linked_spell.md [hunk 12]
```diff
@@ -6,70 +6,17 @@
...
-<td><p> </p></td>
-</tr>
-</tbody>
-</table>
+| Field              | Type         | Attributes | Key | Null | Default | Extra | Comment |
+|--------------------|--------------|------------|-----|------|---------|-------|---------|
+| [spell_trigger][1] | mediumint(8) | signed     |     | NO   | NULL    |       |         |
+| [spell_effect][2]  | mediumint(8) | signed     |     | NO   | 0       |       |         |
+| [type][3]          | smallint(3)  | unsigned   |     | NO   | 0       |       |         |
+| [comment][4]       | text         | signed     |     | NO   | NULL    |       |         |
```
Comment (Kitzunu) [inline docs/spell_linked_spell.md:14]: fix it for all text and varchar.
Cant be signed or unsigned and default can't be NULL if NULL is not allowed, which it is not

File: docs/spell_linked_spell.md [hunk 13]
```diff
@@ -6,70 +6,17 @@
...
-<td><p>NO</p></td>
-<td><p>NULL</p></td>
-<td><p> </p></td>
-<td><p> </p></td>
-</tr>
-</tbody>
-</table>
+| Field              | Type         | Attributes | Key | Null | Default | Extra | Comment |
+|--------------------|--------------|------------|-----|------|---------|-------|---------|
+| [spell_trigger][1] | mediumint(8) | signed     |     | NO   | NULL    |       |         |
```
Comment (Kitzunu) [inline docs/spell_linked_spell.md:11]: Has no standard value so not NULL

File: docs/account_tutorial.md (hunk 1 above)
Comment (pangolp) [reply docs/account_tutorial.md:47]: Thanks

**Comment by pangolp** [comment]
This is how it was in the original files @Kitzunu. I'm not editing the information, I'm just replacing the HTML by Markdown, if not, we do not end more with the pull request, there are even tables, which no longer exist, but anyway I'm editing. Then the next step to review all that you say, because we must modify / update many of the tables, but if not, we do not finish more.

File: docs/creature_formations.md (hunk 4 above)
Comment (pangolp) [reply docs/creature_formations.md:154]: What does this mean? If you explain it to me, I can make sense of it, but the truth is I don't really understand it.

Comment (Kitzunu) [reply docs/creature_formations.md:154]: "If your leader has a path like the one below " it is used to easily understand how it works

Comment (pangolp) [reply docs/creature_formations.md:154]: Ok, now I add it again.

**Comment by pangolp** [comment]
There, the PR is ready for a new revision.

**Comment by FrancescoBorzi** [comment]
@Kitzunu can this be approved?

**Review by Kitzunu — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
thanks @pangolp , and @Kitzunu for the review

**Comment by Kitzunu** [comment]
and @Helias :P

**Comment by pangolp** [comment]
Thank you @Helias and @Kitzunu for correcting my mistakes and giving me more work. I love you all very much. Thank you @FrancescoBorzi for the merger. With this one, I think there are 40 files left to modify.

**Pull request merged**